shell/debug.sh

```diff
@@ -13,7 +13,7 @@ export CGO_ENABLED=1
 set -ex
 
 if [[ -n ${DLV_PORT} ]]; then
-  exec "$SCRIPTS_DIR/gobin.sh" github.com/go-delve/delve/cmd/dlv@v"$(get_application_version "delve")" exec "$(get_repo_directory)/bin/$(get_app_name)" --headless --listen=":${DLV_PORT}"
+  exec "$SCRIPTS_DIR/gobin.sh" github.com/go-delve/delve/cmd/dlv@v"$(get_application_version "delve")" exec "$(get_repo_directory)/bin/${DEV_CONTAINER_EXECUTABLE:-$(get_app_name)}" --headless --listen=":${DLV_PORT}"
```

Comment:
another dumb question:
that expansion definitely works?
I can't do a contrived version :/
```
 ab= 1 echo "${$(echo a)$(echo b)}"
 # bad substitution
 ```

Reply:
One is in curlies

Reply:
this says, do `$DEV_CONTAINER_EXECUTABLE` if set, otherwise do `$(get_app_name)`

Reply:
in your case the equivalent would be

```sh
a=1 
b=2 
echo "${a:-$(echo $b)}"
```

Reply:
ah. i missed the hyphen.

boy, bash-style parameter expansion is something i look up probably once a month. can never keep all the variations straight